dev/manual_tests/lib/text.dart

```diff
@@ -666,7 +663,10 @@ class _FallbackState extends State<Fallback> {
                   ),
                   child: IntrinsicWidth(
                     child: ListBody(
-                      children: lines,
+                      children: <Widget>[
+                        for (String font in androidFonts)
+                          Text(multiScript, style: style.copyWith(fontFamily: font, fontSize: math.exp(_fontSize))),
```

Comment:
Should these be broken out over multiple lines?

Reply:
Done